packages/cli/commands/import/credentials.ts

```diff
@@ -21,7 +21,7 @@ import {
 } from 'n8n-workflow';
 
 import * as fs from 'fs';
-import * as glob from 'glob-promise';
+import * as fg from 'fast-glob';
```

Comment:
maybe let's keep it as `glob`, so we don't need to touch the code

```suggestion
import * as glob from 'fast-glob';
```

Reply:
Thanks for the suggestion, @BHesseldieck